specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json

```diff
@@ -3258,6 +3258,76 @@
           }
         }
       }
+    },
+    "/providers/Microsoft.ADHybridHealthService/services/{serviceName}/reports/riskyIp/GetAllBlobUri": {

```

Comment:
GetAllBlobUri is not an appropriate resource type name, as it implies an action and is not plural. A better example would be: `/reports/riskyIp/blobUris`